src/sentry/static/sentry/app/components/forms/form.jsx

```diff
@@ -71,7 +71,7 @@ export default class Form extends React.Component {
     let curData = this.state.data;
     let newData = {};
     Object.keys(curData).forEach(k => {
-      if (data.hasOwnProperty(k)) newData[k] = data[k];
+      if (data && data.hasOwnProperty(k)) newData[k] = data[k];
```

Comment:
nit: would prefer to avoid executing the loop entirely if data isn't defined

Reply:
You have to run the loop for line below?

Reply:
I mean we could do `if (!data) newData = curData` rather than run through each object property individually

Reply:
Let me know if that looks OK?